tests/util/mod.rs

```diff
@@ -137,6 +137,20 @@ impl<'a> TestManager<'a> {
 
         Ok(())
     }
+
+    /// Remove a temporary table from the list of temporary tables.
+    pub fn remove_temporary_table(
+        &mut self,
+        table_name: String,
+    ) -> Result<(), Box<dyn std::error::Error>> {
+        if self.temporary_tables.contains(&table_name) {
+            self.temporary_tables.retain(|s| !s.contains(&table_name));
```

Comment:
I think the following code is more suitable for this case.

```suggestion
            self.temporary_tables.retain(|s| s != &table_name);
```

Reply:
I agree